modules/auxiliary/scanner/postgres/postgres_login.rb

```diff
@@ -80,6 +95,16 @@ def run_host(ip)
         create_credential_login(credential_data)
 
         print_good "#{ip}:#{rport} - Login Successful: #{result.credential}"
+
+        if create_session?
+          begin
+            postgresql_client = result.proof
```

Comment:
Who's closing the sockets here for the scenario that we don't want to keep the connection open? 🤔